.CI/Test/Common.c

```diff
@@ -13,6 +14,23 @@ char* ModelicaAllocateStringWithErrorReturn(size_t len) {
     return malloc(len + 1); /* Never free'd in the test programs */
 }
 
+char* ModelicaDuplicateString(const char* str) {
+    void *data = malloc(strlen(str) + 1); /* Never free'd in the test programs */
+    assert(data);
+    if (NULL != data) {
+        strcpy(data, str);
+    }
```

Comment:
As far as I can tell, we're already relying on `assert` not being disabled, so the difference to `ModelicaDuplicateStringWithErrorReturn` could be made more clear by removing the redundant guard:
```suggestion
      strcpy(data, str);
```

Reply:
Sure, it is all about test code only.